src/System.Net.Security/src/System/Net/Security/CipherSuitesPolicy.Linux.cs

```diff
@@ -0,0 +1,217 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Security.Authentication;
+using System.Text;
+using Ssl = Interop.Ssl;
+using OpenSsl = Interop.OpenSsl;
+
+namespace System.Net.Security
+{
+    public sealed class CipherSuitesPolicy
```

Comment:
Since this type is public, it should be broken up into the public portion and a PAL portion.  The public portion should call into the PAL, but shouldn't have any platform-specific code in it.